pyfr/integrators/std/steppers.py

```diff
@@ -209,6 +239,7 @@ def step(self, t, dt):
 class StdRK34Stepper(StdRKVdH2RStepper):
     stepper_name = 'rk34'
     stepper_order = 3
+    stepper_has_variable_dt = True
```

Comment:
This is a tad misleading as RK34 can be run quite happily without adaptive step size control (and likely will do a better job than the TVD-RK3 scheme in the process due to its larger stability region).  Probably better to set this in the controller rather than the stepper.